File: python/BOJ_1110.py
N = int( input() )
newN = N
count = int(0)
while True:
    N2 = newN // 10 + newN % 10
    newN = int( str( newN % 10 ) + str( N2 % 10 ) )
    count += 1
    if( N == newN ):
        print( count )
        break

File: java/BOJ_1110.java
import java.util.*;
 
public class BOJ_1110 {
 
    public static void main(String[] args) {
        
        Scanner input = new Scanner(System.in);
        
        int N = input.nextInt();
        int sum = N;
        int count = 0;
        
        while(true) {
            if(sum < 10) {
                count ++;
                sum = sum + sum * 10;
            } else {
                count++;
                sum = ((sum / 10 + sum % 10) % 10) + ((sum % 10) * 10);
            }
            if(sum == N)
                break;
        }
        System.out.println(count);
        
    }
    
}
